### DISCUSSION TITLE
Parallel Debugging MOOSE in VS Code

### DISCUSSION THREAD

**Discussion by Flolaffel** [idaholab/moose#27742] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hello,

I'm currently working on getting my MOOSE Application to run with MPI. It didn't run out of the box, so I'd like to debug in parallel. I read the [MOOSE documentation](https://mooseframework.inl.gov/application_development/debugging.html) on debugging and got parallel debugging from terminal to work. However I'd like to parallel debug in VS Code and am not really sure where to start. 

Is there a common workflow for this?

**Comment by GiudGiud** [comment]
Hello

No we have not set that up yet. 

If you figure out how to do it please let us know

Guillaume
